utils/triggerhappy/patches/010-no-pod.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile
++++ b/Makefile
+@@ -21,7 +21,7 @@ THCMD_COMPS := th-cmd cmdsocket
+ 
+ MAKEDEPEND = $(CC) -M -MG $(CPPFLAGS) -o $*.d $<
+ 
+-all: thd th-cmd man
++all: thd th-cmd
```

Comment:
There should be created feature request in triggerhappy to have option to compile it without docs instead of patching it. In any case, the patch needs some love, I mean commit description, etc.

Reply:
It's possible to avoid patching by defining a custom Build/Compile I think.

Reply:
Well now look what I found...

```
staging_dir/hostpkg/usr/bin/pod2man
```

It's not failing anymore.

Reply:
OTOH having a dependency on perl/host is kind of nasty...